src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[],
+  setFilterStatus: (filter: Filter) => void,
+  isCompleted: boolean,
+  filterStatus: Filter,
+  handleDelete: (id: number) => void,
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  setFilterStatus,
+  isCompleted,
+  filterStatus,
+  handleDelete,
+}) => {
+  const activeTodos = [...todos].filter(todo => !todo.completed);
+  const handleOnClick = (status: Filter) => {
```

Comment:
1. don't use the prefix `handle`
2. the name of the function makes it unclear what it does. therefore, it is better to rename it `chooseFilter`